zio-http/src/main/scala/zio/http/Middleware.scala

```diff
@@ -398,6 +399,89 @@ object Middleware extends HandlerAspects {
   def serveResources(path: Path)(implicit trace: Trace): Middleware[Any] =
     toMiddleware(path, StaticServe.fromResource)
 
+  /**
+   * Creates a middleware for serving the Swagger UI at the given path.
+   *
+   * Example: {{{ val routes: Routes[Any, Response] = ??? val openAPIv1: OpenAPI
+   * \= ??? val openAPIv2: OpenAPI = ??? val swaggerUI =
+   * Middleware.swaggerUI("docs" / "openapi", openAPIv1, openAPIv2) val
+   * routesWithSwagger = routes @@ swaggerUI }}}
+   *
+   * With this middleware in place, a request to `https://www.domain.com/[path]`
+   * would serve the Swagger UI. The different OpenAPI specifications are served
+   * at `https://www.domain.com/[path]/[title].json`. Where `title` is the title
+   * of the OpenAPI specification and is url encoded.
+   */
+  def swaggerUI(path: PathCodec[Unit], api: OpenAPI, apis: OpenAPI*): Middleware[Any] =
+    swaggerUI(path, "5.10.3", api, apis: _*)
```

Comment:
What is this hard-coded version here?

Reply:
It is just a default version, that is the latest for now. I thought it might be nice to not have to look up a swagger ui version for a simple use case.